Commandant/ArgumentParser.swift

```diff
@@ -86,6 +86,15 @@ public final class ArgumentParser {
 		}
 	}
 
+	private init(rawArguments: [RawArgument]) {
```

Comment:
If we make `ArgumentParser` `struct`, option parsing using `<|` operator does not work. Consuming arguments is done in only in a operation and isn't propagated between each operations.